gradio/components/audio.py

```diff
@@ -328,8 +328,8 @@ def get_masked_inputs(self, tokens, binary_mask_matrix):
         return masked_inputs
 
     def postprocess(
-        self, y: tuple[int, np.ndarray] | str | Path | None
-    ) -> str | dict | None:
+        self, y: tuple[int, np.ndarray] | str | Path | bytes | None
+    ) -> str | dict | bytes | None:
```

Comment:
Nice! We should add to the docstring of `Audio` that the returned audio can be in `bytes` format. (Perhaps we even mention that this is the recommended format when streaming). Also, let's add `stream_audio_out` to the list of demos